components/layer-legend.vue

```diff
@@ -18,7 +18,7 @@
     </svg>
     <div class="layer-legend__range unselectable">
       <div v-if="!editingRange" class="layer-legend__range-min">
-        <ui-button kind="quiet" @click="editRange">{{ minValue }}{{ unit }}</ui-button>
```

Comment:
Also here, why remove the unit?

Reply:
The unit was show twice: 
- Firstly, below the color bar: {{ minValue }}{{ unit }} and {{ maxValue }}{{ unit }}
- Secondly, also on the right hand side of the color bar.
See for example:
![image](https://user-images.githubusercontent.com/50665134/106144052-f2f1d380-6173-11eb-90b1-a81cfbf4736a.png)

This is not required. So either the units shown below the color bar, or the unit shown on the right hand side, could be removed. Opted to remove the unit below the color bar because of this.

Although this change is not essential, this was commented on by on the parties providing the new datasets.